packages/terra-filter-view/src/FilterView.jsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import 'terra-base/lib/baseStyles';
+import { Variants } from './_constants';
+import Select from './_Select';
+
+const propTypes = {
+  /**
+   * The dropdown/result menu options.
+   */
+  children: PropTypes.node,
+  /**
+   * The default selected value.
+   */
+  defaultValue: PropTypes.oneOfType([PropTypes.string, PropTypes.number, PropTypes.array]),
+  /**
+   * Whether the select is disabled.
+   */
+  disabled: PropTypes.bool,
+  /**
+   * When true, will disable the auto-search.
+   */
+  disableAutoSearch: PropTypes.bool,
+  /**
+   * Additional attributes to spread onto the dropdown/list. ( style, classnames, etc.. )
+   */
+  dropdownAttrs: PropTypes.shape({}),
+  /**
+   * The minimum number of characters to perform a search.
+   */
+  minimumSearchTextLength: PropTypes.number,
+  /**
+   * Content to display when no results are found.
+   */
+  noResultContent: PropTypes.node,
+  /**
+   * Callback function triggered when the value changes. function(value)
+   */
+  onChange: PropTypes.func,
+  /**
+   * Callback function triggered when an option is selected. function(value, option)
+   */
+  onSelect: PropTypes.func,
+  /**
+   * A callback to indicate an invalid search. function(value)
+   */
+  onInvalidSearch: PropTypes.func,
+  /**
+   * Callback function triggered when the search criteria changes. function(searchvalue)
+   */
+  onSearch: PropTypes.func,
+  /**
+   * Callback function invoked for each option on search change. function(searchvalue, option)
+   */
+  optionFilter: PropTypes.func,
+  /**
+   * Placeholder text.
+   */
+  placeholder: PropTypes.string,
+  /**
+   * Controls whether or not all results are shown on initial load or after input focus.
+   */
+  showResultsInitially: PropTypes.bool,
+  /**
+   * How long the component should wait (in milliseconds) after input before performing an automatic search.
+   */
+  searchDelay: PropTypes.number,
+  /**
+   * The search field variant. One of `dropdown` or `list`
+   */
+  variant: PropTypes.oneOf([
+    Variants.DROPDOWN,
+    Variants.LIST,
+  ]),
+};
+
+const defaultProps = {
+  children: undefined,
+  defaultValue: undefined,
+  disabled: false,
+  disableAutoSearch: false,
+  dropdownAttrs: undefined,
+  minimumSearchTextLength: 2,
+  noResultContent: undefined,
+  onChange: undefined,
+  onInvalidSearch: undefined,
+  onSearch: undefined,
+  onSelect: undefined,
+  optionFilter: undefined,
+  placeholder: undefined,
+  searchDelay: 250,
+  showResultsInitially: true,
```

Comment:
Booleans should default to false otherwise a user is required to override a lint error to satisfy `showResultsInitially={false}`

This could be changed to `hideResultsInitially` to satisfy the requirements of showing the results initially but a true flag would hide them.